Formula/aom.rb

```diff
@@ -0,0 +1,44 @@
+class Aom < Formula
+  desc "Codec library for encoding and decoding AV1 video streams"
+  homepage "https://aomedia.googlesource.com/aom"
+  url "https://aomedia.googlesource.com/aom", :using => :git, :tag => "v1.0.0", :revision => "d14c5bb4f336ef1842046089849dee4a301fbbf0"
+
+  depends_on "cmake" => :build
+  depends_on "yasm" => :build
+
+  def install
+    args = %w[
+      -DENABLE_EXAMPLES=on
+      -DENABLE_DOCS=off
+      -DENABLE_TESTDATA=off
+      -DENABLE_TESTS=off
+      -DENABLE_TOOLS=off
+    ]
+
+    mkdir "macbuild" do
+      system "cmake", "..", *std_cmake_args, *args
```

Comment:
Inline `args` instead of creating an array

Reply:
Done.